Cargo.toml

```diff
@@ -31,6 +31,10 @@ quote = "1.0.28"
 # Errors
 thiserror = "1.0.40"
 
+#logs 
```

Comment:
It's a nitpick, but please format like everything else or things just look clunky